jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -155,6 +155,22 @@ public static void copy(Source source, Sink sink, Chunk.Processor chunkProcessor
      */
     public interface Source
     {
+        /**
+         * Factory of {@link Content.Source}.
+         */
+        interface Factory
+        {
+            /**
+             * Creates a new {@link Content.Source}.
+             *
+             * @param sizedBufferPool the {@link ByteBufferPool.Sized} to get buffers from. {@code null} means allocate new buffers as needed.
```

Comment:
Allow null to be passed for non-pooling buffer of a default size

Reply:
The contract explicitly allows that. Is the wording unclear?

Reply:
for some reason I did not see it ?